ibc_data/conditions.csv

```diff
@@ -0,0 +1,377 @@
+RSVPLanguage,jabberwocky,syntactically congruent sentences composed by non-lexical vocable constituents
+RSVPLanguage,word_list,syntactically non-congruent sentences but with semantic content
+RSVPLanguage,pseudoword_list,syntactically and semantically non-congruent sentences composed by non-lexical vocable constituents
+RSVPLanguage,consonant_string,syntactically and semantically non-congruent sentences composed by non-vocable constituents
+RSVPLanguage,probe,
+MTTWE,we_average_reference,action in the story to serve as reference for the time or space judgment in the same trial in the west-east island
+MTTWE,we_all_space_cue,cue indicating a question about spatial orientation in the west-east island
+MTTWE,we_all_time_cue,cue indicating a question about time orientation in the west-east island
+MTTWE,we_westside_event,"action to be judged whether it takes place west or east from this reference, that actually takes place west from this reference"
+MTTWE,we_eastside_event,"action to be judged whether it takes place west or east from this reference, that actually takes place east from this reference"
+MTTWE,we_before_event,"action to be judged whether it takes place before or after this reference, that actually takes place before this reference, in the west-east island"
+MTTWE,we_after_event,"action to be judged whether it takes place before or after this reference, that actually takes place
+before this reference, in the west-east island"
+MTTWE,we_all_event_response,motor responses performed after every event condition in the west-east island
+MTTNS,sn_average_reference,action in the story to serve as reference for the time or space judgment in the same trial in the west-east island
+MTTNS,sn_all_space_cue,cue indicating a question about spatial orientation in the south-north island
+MTTNS,sn_all_time_cue,cue indicating a question about time orientation in the south-north island
+MTTNS,sn_southside_event,"action to be judged whether it takes place south or north from this reference, that actually takes place south from this reference"
+MTTNS,sn_northside_event,"action to be judged whether it takes place south or north from this reference, that actually takes place north from this reference"
+MTTNS,sn_before_event,"action to be judged whether it takes place before or after this reference, that actually takes place before this reference, in the south-north island"
+MTTNS,sn_after_event,"action to be judged whether it takes place before or after this reference, that actually takes place before this reference, in the south-north island"
+MTTNS,sn_all_event_response,motor responses performed after every event condition in the south-north island
+PreferenceFood,food_constant,classify the level of pleasantness of a food item displayed on the screen in terms of willingness to eat it. visual evaluation of an item vs. fixation

```

Comment:
Maybe then I should only describe "food", "paintings", etc and get rid of the "constant", "linear", etc...?

Reply:
Please keep them all, they have a a different meaning. We can talk about that tomorrow (please remind me).

Reply:
Added that line of description to each condition! Should I also add it to the descriptions in the documentation?

Reply:
I added the lines in the csv files, however, I am not sure how to match this in the documentation tex file, since the tables are structured differently: there we have a sentence for "food/paint/house/face evaluation" as conditions' description, and in the next table the contrast are described as preference constant/linear/quadratic; not all together as in the csv files. My idea to match this in the tex file is to add this "constant is an occurrence regressor..." sentence to the constrasts' description and not modify the structure of the tables.

Reply:
Yes, please do. Thx.